helper/escapingio/reader_test.go

```diff
@@ -426,14 +426,16 @@ func naiveEscapeCharacters(input string, escapeChar byte, h Handler) string {
 	}
 
 	return reg.ReplaceAllStringFunc(input, func(match string) string {
-		if len(match) != 3 {
-			panic(fmt.Errorf("match isn't 3 characters: %s", match))
+		// match can be more than three bytes because of unicode
```

Comment:
if the intention is to support unicode, could we switch this to `utf8.RuneCountInString`?

Reply:
Nope - we meant to only process ascii characters for escaping purposes.